server/src/main/java/com/cloud/deploy/FirstFitPlanner.java

```diff
@@ -367,8 +372,11 @@ protected void removeClustersCrossingThreshold(List<Long> clusterListForVmAlloca
                 // Remove clusters crossing disabled threshold
                 clusterListForVmAllocation.removeAll(clustersCrossingThreshold);
 
-                s_logger.debug("Cannot allocate cluster list " + clustersCrossingThreshold.toString() + " for vm creation since their allocated percentage" +
-                        " crosses the disable capacity threshold defined at each cluster/ at global value for capacity Type : " + capacity + ", skipping these clusters");
+                String warnMessageForClusterReachedCapacityThreshold = String.format(
+                        "Cannot allocate cluster list %s for vm creation since their allocated percentage crosses the disable capacity threshold defined at each cluster at"
```

Comment:
CLGTM, just have one suggestion:


```suggestion
                        "Cannot allocate cluster list %s for VM creation since their allocated percentage crosses the disable capacity threshold defined at each cluster at"
```

Reply:
Thanks @GutoVeronezi. Updated it!